subjects/printandmiss/README.md

```diff
@@ -0,0 +1,49 @@
+## printandmiss
+
+### Instructions
+
+Write a function called `PrintAndMiss()` that takes a `string` and an `int` as an argument. The function should move through the `string` in sets determined by the `int`, printing the first set, omitting the second, printing the third, and so on, in a 'print' and 'miss' fashion until the end of the `string` is reached. Return a `string` containing the printed characters.
```

Comment:
Having a function named print, and then asking to return is contradictory. If you want to evaluate the result by returning, please change the name of the subject.

Reply:
changed to `SaveAndMiss`